topo_processor/metadata/loaders/metadata_loader_imagery_historic.py

```diff
@@ -19,37 +23,39 @@ def is_applicable(self, item: Item) -> bool:
         return True
 
     def add_metadata(self, item: Item) -> None:
-        file_name = os.path.splitext(os.path.basename(item.path))[0]
+        if not self.is_init:
+            self.read_csv()
+        item_basename = os.path.splitext(os.path.basename(item.path))[0]
+        item_metadata = self.csv_dict[item_basename]
+        properties = {
+            "linz:sufi": item_metadata[self.csv_header.index("sufi")],
```

Comment:
is there a way to store these indexes and not re look them up for every line

Reply:
good idea, we will look into that

Reply:
investigate parse as dictionary / array in the csv lib